salt/cloud/clouds/joyent.py

```diff
@@ -281,6 +281,7 @@ def create(vm_):
     salt.utils.cloud.check_name(vm_['name'], 'a-zA-Z0-9-.')
     kwargs = {
         'name': vm_['name'],
+        'networks': vm_['networks'],
```

Comment:
This should be `vm_.get('networks', some_sane_default)` since not everybody will be using this.